workorder-job-repeater.lua

```diff
@@ -0,0 +1,296 @@
+-- reduces pauses between jobs of infinite work orders
+
+local function print_help()
+    print[====[
+
+workorder-job-repeater
+======================
+This script recreates workshop jobs upon completion in case
+they come from a work order with amount set to infinite. Usually
+a job for an infinite work order is only created once a day,
+resulting in dwarfs lazying around (this doesn't apply to non-infinite
+work orders). This script makes the dwarfes work harder.
+
+For best experience add following to your ``onMapLoad*.init``::
+
+    workorder-job-repeater start
+    
+**Usage**:
+
+``[enable | disable] workorder-job-repeater``
+
+``workorder-job-repeater start [<frequency>] | stop``
+
+``workorder-job-repeater --frequency <ticks> | --verbose | --help``
+
+:enable, start <frequency>: starts job monitoring, running every ``<frequency>`` ticks (default: 10).
+:disable, stop:             stops the script.
+
+-f, --frequency <ticks>     sets job monitoring frequency to ``<ticks>``
```

Comment:
How is a user to decide what tick frequency to use? Does this need to be configurable?

Reply:
the more frequent the better, but more CPU intensive. I'd like to leave this as an option just in case.

Reply:
fair enough. maybe just document the default here so users know the flag is optional.

Reply:
did that